README.md

```diff
@@ -40,6 +40,10 @@ make install
    options. Some of the options are generic autoconf options, while the sockperf
    specific options are prefixed with "SOCKPERF:" in the help text.
    
+ * To enable unit tests
```

Comment:
do you mean `enable TLS support`

Reply:
I'm following the same format as in other bullets below